src/Cake.Core/IO/NuGet/NuGetPackageInstaller.cs

```diff
@@ -31,8 +31,8 @@ public void InstallPackage(NuGetPackage package, DirectoryPath installationRoot)
             var process = _processRunner.Start(nugetPath, new ProcessSettings
             {
                 Arguments = GetNuGetToolInstallArguments(package, installationRoot),
-                RedirectStandardOutput = true,
-                Silent = true
+                RedirectStandardOutput = false,
+                Silent = false
```

Comment:
@gep13 That change was unintentional. Was outputting NuGet information
while testing. Will change back. Thanks for seeing this :smile:
On Jan 6, 2016 08:54, "Gary Ewan Park" <notifications@github.com> wrote:

> In src/Cake.Core/IO/NuGet/NuGetPackageInstaller.cs
> <https://github.com/cake-build/cake/pull/617#discussion_r48934436>:
>
> > @@ -31,8 +31,8 @@ public void InstallPackage(NuGetPackage package, DirectoryPath installationRoot)
> >              var process = _processRunner.Start(nugetPath, new ProcessSettings
> >              {
> >                  Arguments = GetNuGetToolInstallArguments(package, installationRoot),
> > -                RedirectStandardOutput = true,
> > -                Silent = true
> > +                RedirectStandardOutput = false,
> > +                Silent = false
>
> @patriksvensson <https://github.com/patriksvensson> why did this change?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cake-build/cake/pull/617/files#r48934436>.
>